conans/paths/package_layouts/package_cache_layout.py

```diff
@@ -136,3 +138,28 @@ def package_summary_hash(self, pref):
         package_folder = self.package(pref)
         readed_manifest = FileTreeManifest.load(package_folder)
         return readed_manifest.summary_hash
+
+    # Raw access to file
+    def get_path(self, path, package_id=None):
```

Comment:
```suggestion
    def get_content(self, path, package_id=None):
```
Since you are returning the content for given path